### PR TITLE
Pager

### PR DESCRIPTION
**Pull request by rvrita**
PR for pagination. Notes below.

There are some TODO comments in the code, trying to avoid this PR to be bigger (will come back to those later).
Styled components refactor is TODO. 
The pagination feature is not 100% complete yet - it's missing the logic for hiding the page numbers, it just lists them all. 
Instead of this:
Screen Shot 2020-08-25 at 3 12 21 AM
Looks like this:
Screen Shot 2020-08-25 at 3 08 29 AM

Also added the logic for dynamically pulling data from db based on product id in the URL.

### CODE REVIEW

File: client/components/Pager.jsx [hunk 1]
```diff
@@ -1,19 +1,132 @@
...
+    return <Link onClick={onClick} href={href}>{children}</Link>;
+  }
+  return children;
+}
 
 class Pager extends React.Component {
   render() {
+    const pages = [];
+    const { props } = this;
+    for (let i = 1; i < props.totalPages + 1; i += 1) {
```
Comment (ttreit) [inline client/components/Pager.jsx:64]: Did you mean this to be i+= instead of i++?

File: client/components/Pager.jsx [hunk 2]
```diff
@@ -1,19 +1,132 @@
...
+                  props.handlePageClick(props.activePage - 1);
+                },
+              },
+                <Span>
+                  <svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+                    <path d="M6.7 11.3L6 12l.7.7 4 4c.4.4 1 .4 1.4 0 .4-.4.4-1 0-1.4L9.8 13H17c.6 0 1-.4 1-1s-.4-1-1-1H9.8l2.3-2.3c.2-.2.3-.4.3-.7 0-.6-.4-1-1-1-.3 0-.5.1-.7.3l-4 4z" />
+                  </svg>
+                </Span>)}
+            </ListItem>
+            {pages.map((pagenum) => (
```
Comment (ttreit) [inline client/components/Pager.jsx:85]: camel case pagenum (pageNum) to make it easier to read or is this the correct convention for this?

File: client/components/Pager.jsx (hunk 1 above)
Comment (rvrita) [reply client/components/Pager.jsx:64]: Eslint 🙄  😅

File: client/components/Pager.jsx (hunk 2 above)
Comment (rvrita) [reply client/components/Pager.jsx:85]: Good catch!!! Will fix it

**Pull request merged**